softwareComponents/atoms/include/atoms/util.hpp

```diff
@@ -117,11 +117,49 @@ class Defer {
  * Decide if a value fits given type
 */
 template < typename T >
-constexpr bool fitsIn(auto x) {
```

Comment:
Nechceš použít `if constexpr`?

Reply:
Přišlo mi idiomatické se tvářit, že je to přetížení pro různé případy. Spíše jsem uvažoval jestli `static_asserty` nevytknout do společné wrapper funkce.

Reply:
mně by přišlo idiomatické se tvářit, že to je jedna funkce, co se jenom podle typu chová různě (asi jako libovolná templatovaná funkce)
... hlavně proto, že stačí jedna dokumentace, je ten kód na jednom místě a při pohledu vím, že jsem pokryl všechny případy a že se nepřekrývají

ale asi jak chceš, netrvám u tohoto na ničem

Reply:
Ok, tvé argumenty mě přesvědčily. Změněno.